libimage/copier.go

```diff
@@ -472,6 +485,22 @@ func (c *Copier) Copy(ctx context.Context, source, destination types.ImageRefere
 	return returnManifest, retry.IfNecessary(ctx, f, &c.retryOptions)
 }
 
+func (c *Copier) copyToStorage(ctx context.Context, source, destination types.ImageReference) (*storage.Image, error) {
+	var reportResolvedReference types.ImageReference
```

Comment:
Non-blocking:
```suggestion
	var resolvedReference types.ImageReference
```
would be a tiny bit less confusing.